### PR TITLE
[11.0][MIG] save_translation_file: Migration to 11.0

### PR DESCRIPTION
**Pull request by oihane**
from version 8.0 to 11.0

### CODE REVIEW

File: save_translation_file/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    "depends": [
+        "base",
+    ],
+    "data": [
+        "views/save_translation_file_view.xml",
+    ],
+    "demo": [
+    ],
+    "qweb": [
```
Comment (fkantelberg) [inline save_translation_file/__manifest__.py]: Please avoid empty keys. qweb, demo and external_dependencies are looking empty

**Review by fkantelberg — CHANGES_REQUESTED** [review]
LGTM

File: save_translation_file/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "name": "Save translation file",
+    "summary": "Allows developers to easily generate i18n files",
+    "version": "11.0.1.0.0",
+    "category": "Tools",
+    "website": "https://github.com/OCA/server-tools",
+    "author": "Akretion, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "external_dependencies": {
```
Comment (tbaden) [inline save_translation_file/__manifest__.py]: please remove empty tags

File: save_translation_file/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        "python": [],
+        "bin": [],
+    },
+    "depends": [
+        "base",
+    ],
+    "data": [
+        "views/save_translation_file_view.xml",
+    ],
+    "demo": [
```
Comment (tbaden) [inline save_translation_file/__manifest__.py]: please remove empty tags

File: save_translation_file/__manifest__.py (hunk 1 above)
Comment (tbaden) [inline save_translation_file/__manifest__.py]: please remove empty tags

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for your work 👍 
please squash your commits.
and some minor changes

**Pull request closed without merging**